sql/adhoc.sql

```diff
@@ -187,7 +187,8 @@ SELECT
         SUM(settle_amount)::numeric as total,
         DATE(timestamp) as day
       FROM stripe
-      WHERE timestamp > '2015-11-01'
+      WHERE timestamp > '2015-10-04T07:00:00.000Z'
```

Comment:
One thing I haven't looked at within postgres is how to shift the timezone to PST; right now it's all UTC/GMT